media-libs/tiff/files/tiff-4.5.0-CVE-2023-0795,CVE-2023-0796,CVE-2023-0797,CVE-2023-0798,CVE-2023-0799.patch

```diff
@@ -0,0 +1,191 @@
+Index: tiff-4.5.0/tools/tiffcrop.c
```

Comment:
For these, could you append .patch to the URL, then put the contents in the file? That way we get extra patch metadata (author/commit message).

Keep the links in the patch as well like you did though.

Reply:
Could you rename the patch to something like tiff-4.5.0-CVE-2023.patch or similar? I know it's a bit janky, but I'm a bit worried about the portability of commas in filenames on some filesystems.

Reply:
Finally, could you file a bug on Bugzilla under Gentoo Security > Vulnerabilities? You don't need to worry about filling it out in detail, just include the CVEs in the Aliases field (comma-separated).

Reply:
Thanks for the quick feedback!

>  For these, could you append .patch to the URL, then put the contents in the file? 

The problem is that one of the commits (https://gitlab.com/libtiff/libtiff/-/commit/9c22495e5eeeae9e00a1596720c969656bb8d678) is not directly linked with a CVE. So basically we need commits to fix one security root problem, thus I put them into one file. So if I put this one change in a separate file what should I name it?

> Could you rename the patch to something like tiff-4.5.0-CVE-2023.patch or similar?

In this case its like one root problem that fixes 5 CVEs. So my intention was to mark all the fixed CVEs in the patch name. Should I mention only one and discard the others that are also fixed? Or should I just append the CVEs without comma separation?

> Finally, could you file a bug on Bugzilla under Gentoo Security > Vulnerabilities?

Will do :)

Reply:
>The problem is that one of the commits ([gitlab.com/libtiff/libtiff/-/commit/9c22495e5eeeae9e00a1596720c969656bb8d678](https://gitlab.com/libtiff/libtiff/-/commit/9c22495e5eeeae9e00a1596720c969656bb8d678)) is not directly linked with a CVE. So basically we need commits to fix one security root problem, thus I put them into one file. So if I put this one change in a separate file what should I name it?

This part is okay, just concatenate them (so wget or whatever the first patch, then the second appended).

Reply:
Showing for future purposes, not as a "how did you miss this" (BZ's interface is really kind of busy and it's easy to miss stuff):
<img width="1365" alt="Screenshot 2023-02-22 at 16 10 23" src="https://user-images.githubusercontent.com/11667869/220685861-aedf4ed7-02f6-4389-97ba-d96961608700.png">

It's the bit on the left under Status.

Reply:
> This part is okay, just concatenate them

I changed the patches and hope I understood you correctly.

> Showing for future purposes

Thanks :)